oap-server/server-configuration/configuration-nacos/pom.xml

```diff
@@ -45,4 +45,143 @@
             <artifactId>nacos-client</artifactId>
         </dependency>
     </dependencies>
+
```

Comment:
Here we also bring the Nacos dynamic configuration IT back, to see whether it works. #2841

Reply:
Yes. Please consider my suggesstion, isolate ITtest from travis and appveyor

Reply:
Done, thanks for your advice